crates/compilers/src/lib.rs

```diff
@@ -882,6 +889,21 @@ fn rebase_path(base: &Path, path: &Path) -> PathBuf {
     new_path.to_slash_lossy().into_owned().into()
 }
 
+/// Utility function to change source content ranges with provided updates.
+fn replace_source_content(source: &str, updates: impl Iterator<Item = Update>) -> String {
```

Comment:
this shouldnt use Update, should be Range + &str

Reply:
changed in https://github.com/foundry-rs/compilers/pull/252/commits/d37cafb8f47521c4e50b126db2acac6dcd4e002a